### DISCUSSION TITLE
eslint not registered as command after install

### DISCUSSION THREAD

**Discussion by MMK21Hub** [eslint/eslint#13686] [General]
**Tell us about your environment**
```
eslint : The term 'eslint' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the 
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ eslint --env-info
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (eslint:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```
* **ESLint Version:** ?
* **Node Version:** v12.18.3
* **npm Version:** 6.14.6

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
?
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
//
```

```bash
eslint --init 
```

**What did you expect to happen?** It to do stuff

**What actually happened? Please include the actual, raw output from ESLint.**
```
eslint : The term 'eslint' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the 
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ eslint --init
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (eslint:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

**Are you willing to submit a pull request to fix this bug?** No

**Comment by kaicataldo** [comment]
Have you installed eslint before running the above command? If not, you'll need to use `npx eslint --init` to initiate the configuration wizard.

**Reply by KadirZukorlic** [reply]
This helped. thx a lot!

**Comment by MMK21Hub** [comment]
Still get the same error after going through the wizard.

**Comment by anikethsaha** [comment]
did you install the dependencies ? i.e the last question about whether to install it or not, if not installed, can you try installing them

**Comment by MMK21Hub** [comment]
I think I have the dependencies and am getting the same error after reinstalling.

**Comment by anikethsaha** [comment]
can you check once using `npm list | grep "eslint"` or `npm list | find "eslint"` for windows ?

**Comment by mdjermanovic** [comment]
This is expected for all locally installed packages.

Once you install ESLint, you can run it with `npx eslint`, or directly `.\node_modules\.bin\eslint`. 

There is also [eslint-cli](https://github.com/eslint/eslint-cli) which can be installed globally and provides `eslint` command that runs locally installed ESLint.

**Comment by juanvictorbascopecastro** [comment]
Error
![image](https://user-images.githubusercontent.com/43118668/182040596-04eb94ab-0359-4f4e-8c00-87c12160f05f.png)
